packages/xo-web/src/common/select-objects.js

```diff
@@ -166,40 +202,26 @@ class GenericSelect extends React.Component {
             options.push(getOption(object, container))
           })
         })
-      }
 
-      const objectsById = this._getObjectsById()
-      const addIfMissing = val => {
-        if (val != null && !(val in objectsById)) {
-          options.push({
-            disabled: true,
-            id: val,
-            label: val,
-            value: val,
-            xoItem: {
-              id: val,
-              removed: true,
-            },
+        // missing objects don't have a container
```

Comment:
What about giving them a "Missing objects" special container?

Reply:
Just what I did in the `_getObjects`